setup.py

```diff
@@ -6,7 +6,7 @@
 readme = f.read().strip()
 
 f = open('LICENSE')
-license = f.read().strip()
+license_content = f.read().strip()
```

Comment:
Renamed to avoid conflict with built-in constant. https://docs.python.org/3.7/library/constants.html#license